src/expression.d

```diff
@@ -4829,12 +4829,12 @@ public:
      * Convert string contents to a 0 terminated string,
      * allocated by mem.xmalloc().
      */
-    final const(char)* toStringz() const
+    extern (D) final const(char)[] toStringz() const
```

Comment:
In Walter's mind, the mismatch between D and C++ code is less important issue than frustration about code style.
Refer: #5751.